opensciencegrid/ospool-cm/Dockerfile

```
@@ -26,15 +27,21 @@ RUN if [[ $BASE_YUM_REPO = release ]]; then \
       yum -y install condor; \
     fi
 
+# basic config is a collector, so we can test
+RUN echo "DAEMON_LIST = MASTER, SHARED_PORT, COLLECTOR" >/etc/condor/config.d/05-ospool-base.config
```

```suggestion
RUN echo "DAEMON_LIST = MASTER, COLLECTOR" >/etc/condor/config.d/05-ospool-base.config && \
    echo "USE_SHARED_PORT = TRUE"         >>/etc/condor/config.d/05-ospool-base.config
```
although isn't shared port the default?